content/blog/posts/reasons-to-go-to-grad-school.md

```diff
@@ -125,17 +125,17 @@ place.
 Unlike a PhD, a Master's program is a great place to avoid being an adult for
 a year or two, especially with the current explosion in undergraduate
 enrollment in computer science. Many undergraduate programs are short on
-staff, including on teaching assistants, so if you can get a free Master's
-degree by working as a teaching assistant, and you want another year of
+staff---including on teaching assistants---so if you can get a free Master's
```

Comment:
This was a really long sentence with a lot of commas, and I wanted to break it up _somehow_.